src/main/resources/db/migration/V17__POL-453-expose-last-triggered-to-policies.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE policy ADD column last_triggered bigint default 0;
+
+UPDATE policy as p SET last_triggered = (SELECT MAX(ctime) FROM policies_history as ph WHERE p.id = uuid(ph.policy_id));
+
+CREATE FUNCTION update_last_triggered() RETURNS TRIGGER AS $update_last_triggered$
+    BEGIN
+        UPDATE policy SET last_triggered = GREATEST(last_triggered, new.ctime) WHERE id = uuid(new.policy_id);
```

Comment:
General comment: be careful with the `policy` PK, it seems it is composed with the `id` and the `customerid` from what I can see in the old migration scripts. Postgres doesn't show it in my local DB though, that's weird.